cli/compose/loader/loader.go

```diff
@@ -555,6 +590,17 @@ func transformServiceSecret(data interface{}) (interface{}, error) {
 	}
 }
 
+func transformServiceConfigObj(data interface{}) (interface{}, error) {
```

Comment:
i think we can use the same function as secrets, since there is no type and the "string" version is the same?